tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -1355,7 +1355,7 @@ TEST_F(EnergyPlusFixture, TestMultiSpeedWasteHeat)
 
     CalcMultiSpeedDXCoilCooling(1, 1, 1, 2, 1, 1, 0);
 
-    EXPECT_NEAR(1303.4304, MSHPWasteHeat, 0.001);
+    EXPECT_NEAR(1302.7482629848564, MSHPWasteHeat, 0.001);
```

Comment:
In general, I think we should include (or *only include*) as many sig-figs in the expected results as we are checking for in the tolerance. So in this case, truncate the expected result to a sig-fig of 1E-3.

Reply:
Thank you for letting me know. I'll fix all unit tests and update the branch again.